### DISCUSSION TITLE
create packages for vlang

### DISCUSSION THREAD

**Discussion by ctkqiang** [vlang/v#14854] [General]
If we would to contribute to the project by creating packages for Vlang, where do we start, and how do we do it?

**Comment by larpon** [comment]
Until we reach 1.0, it's my general understanding that, the team *does not* want to package and distribute V via package managers - the simple reason being that the project has very swift updates for crucial bugs on a daily (sometimes hourly) basis. We've already had cases where users encounters bugs and come to the #help channel on Discord and the solution for them was to update V. So we currently want users to install from source and then do `v up` frequently. After v1.0 it's probably better to look at distribution packaging 📦

**Reply by ctkqiang** [reply]
Until then I am always happy to contribute for building packages like jitsimeet, zoom ,and some other video codec.

**Reply by larpon** [reply]
> Until then I am always happy to contribute for building packages like jitsimeet, zoom ,and some other video codec.

We always appreciate any help we can get! It's just that this specific thing has been working against us because of the development speed 😅 (luxury problem!)

So you're more than welcome to contribute in other ways 🙂

**Comment by ylluminate** [comment]
@johnmelodyme initially I interpreted your meaning as distribution packages for various Linux distros, macOS, Windows, etc... but your other comment regarding video codecs makes me wonder: are you referring to building libs/modules such as those found here, for example?: https://github.com/vlang/awesome-v

**Reply by larpon** [reply]
Good point. 

For clarity: my answer above is in regard to distributing *V itself* via e.g. Linux package managers like `apt`, `pacman` or `homebrew` etc.

**Reply by ctkqiang** [reply]
My bad I wasn't clear enough. It's not a codec but more of a SDK. My intention is to create packages. Your first understanding of my statement is correct, but the second one I wasn't clear.
